next.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path')
+const { execSync } = require('child_process')
+const withCSS = require('@zeit/next-css')
+const withImages = require('next-images')
+const { version } = require('./package.json')
+
+const COMMIT_SHA =
+  process.env.COMMIT_SHA || execSync("git log --pretty=format:'%h' -n 1")
```

Comment:
I wonder if we should use `'%H'`, so `COMMIT_SHA` is always the full SHA, just for consistency's sake

Reply:
Done in https://github.com/aragon/anj.aragon.org/pull/23/commits/6f65b01fdd19c38c0bd686eb3b8d3b8ace0208fb